templates/projects/macros.j2

```diff
@@ -67,17 +67,17 @@
 
         {% if project.case_study %}
           {% call content.link_button(link) %}
-            read our case study »
+            Read our case study
           {% endcall %}
         {% endif %}
         {% if project.url or project.owner %}
           {% call content.link_button(project.url or ['/', page.slug, '/']|join('')) %}
-            view the site »
+            View the site
           {% endcall %}
         {% endif %}
         {% if project.source %}
           {% call content.link_button(project.source) %}
-            source code »
+            Source code
```

Comment:
Maybe this text should match the text above in `templates/layout/macros.j2` in the `rst_meta` macro?